src/html/tags/d/Details.hack

```diff
@@ -14,9 +14,12 @@ use namespace Facebook\XHP\ChildValidation as XHPChild;
 final xhp class details
   extends element
   implements
+    //--- Phrase is not correct, do we keep it for BC?
```

Comment:
Incorrect for this element, or more broadly? Probably remove it

Reply:
These three comments apply to elements which have a category which they don't have in the spec. Removing them would be a bc break.

Especially removing flow from style, since this would make embedding style tags inside the body element invalid. According to a strict interpretation of [the spec document](https://html.spec.whatwg.org/multipage/semantics.html#the-style-element) style inside of the body is disallowed.

I'd say we let style implement flow, since every modern browser will treat it as valid. I left those comments in to (hopefully) spark a discussion about the purpose of child validation. Do we want to enforce the spec or make sure that your html will be understood by browsers and ignore the spec in those cases?

Reply:
We're generally fine with BC breakages in v4; if this turns out to be a huge blocking mistake, we can re-add the category in v4.0.1

let's follow the spec for now, and get more liberal if we need to